mod/node-core/pkg/components/module/depinject.go

```diff
@@ -122,9 +66,43 @@ type DepInjectOutput struct {
 
 // ProvideModule is a function that provides the module to the application.
 func ProvideModule(in DepInjectInput) (DepInjectOutput, error) {
+	// TODO: this is hood as fuck.
+	if in.BeaconConfig.KZG.Implementation == "" {
```

Comment:
I was melding, I think we can just pass an empty runtime during the command part and this solves the problem for now